exercises/darts/canonical-data.json

```diff
@@ -9,44 +9,62 @@
          "property": "score",
          "description": "A dart lands outside the target",
          "input": {
-           "x": 15.3,
-           "y": 13.2
+           "x": -9,
+           "y": 9
          },
          "expected": 0
         },
         {
          "property": "score",
          "description": "A dart lands just in the border of the target",
          "input": {
-           "x": 10,
-           "y": 0
+           "x": 0,
+           "y": 10
          },
          "expected": 1
         },
+        {
+         "property": "score",
+         "description": "A dart lands in the outer circle",
+         "input": {
+           "x": 4,
+           "y": 4
+         },
+         "expected": 1
+        },
+        {
+         "property": "score",
+         "description": "A dart lands right in the border between outer and middle circles",
+         "input": {
+           "x": 5,
+           "y": 0
+         },
+         "expected": 5
```

Comment:
note that here, the dart is treated as being in the smaller circle (here, the middle one) rather than the larger one (outer)